src/test/groovy/liquibase/ext/redshift/LiquibaseHarnessAdvancedSuiteTest.groovy

```diff
@@ -1,11 +1,5 @@
 package liquibase.ext.redshift
 
-import liquibase.harness.AdvancedHarnessSuite
-import liquibase.harness.config.TestConfig
+class LiquibaseHarnessAdvancedSuiteTest extends AdvancedExtensionHarnessSuite {
```

Comment:
AdvancedExtensionHarnessSuite and LiquibaseHarnessAdvancedSuiteTest are the same thing, not need of both